packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -0,0 +1,81 @@
+import { radioName, RadioTokens, RadioSlotProps, RadioProps } from './Radio.types';
+import { UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { defaultRadioTokens } from './RadioTokens';
+
+const radioSize = 20;
+const radioRadius = radioSize / 2;
+
+const radioInnerCircleSize = 10;
+const radioInnerCircleRadius = radioInnerCircleSize / 2;
+
+export const radioSelectActionLabel = 'Select a RadioButton';
+
+export const radioStates: (keyof RadioTokens)[] = ['focused', 'hovered', 'pressed', 'selected', 'disabled'];
+
+export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, RadioTokens> = {
+  tokens: [defaultRadioTokens, radioName],
+  states: radioStates,
+  slotProps: {
+    root: buildProps(() => ({
+      accessible: true,
```

Comment:
accessible, focusable, and accessibilityRole should be props that can be overridden by props passed in by a parent prop, so these should be configured in useRadio.

Also, we try to keep styling files focused on token integration and styling, and accessible, focusable, and accessibilityRole are not considered styles.

Reply:
That makes sense. These props are now configured in useRadio.